packages/aws-cdk-lib/aws-scheduler/README.md

```diff
@@ -1,27 +1,64 @@
-# AWS::Scheduler Construct Library
+# Amazon Scheduler Construct Library
```

Comment:
```suggestion
# Amazon EventBridge Scheduler Construct Library
```

Reply:
although i'm 80% sure we have a linter rule that prevents this, we'll either have to go with AWS::Scheduler or add an exemption in the linter